drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -54,7 +55,19 @@ struct aci_set_tx_power {
 	u8_t value[2];
 };
 
+struct aci_set_ble_addr {
+	u8_t config_offset;
+	u8_t length;
+	u8_t value[6];
+} __packed;
+
 #define ACI_WRITE_SET_TX_POWER_LEVEL       BT_OP(BT_OGF_VS, 0xFC0F)
+#define ACI_HAL_WRITE_CONFIG_DATA	   BT_OP(BT_OGF_VS, 0xFC0C)
+
+#define CONFIG_DATA_PUBADDR_OFFSET		0
```

Comment:
This does not come from kconfig? I think the CONFIG_ prefix might be reserved.

Reply:
No.I think I need to rename this defines

Reply:
fixed